src/MaplibreGLLayer.js

```diff
@@ -168,6 +168,13 @@ export var MaplibreGLJSLayer = Layer.extend({
 
     this._glMap = new maplibregl.Map(options);
 
+    // Listen for style data error (401 Unauthorized)
+    this._glMap.on('error', function (error) {
+      if (error.error && error.error.status === 401) {
+        console.warn('Invalid API key. Please check your ArcGIS API key.');
```

Comment:
```suggestion
        console.warn('Invalid or expired API key. Please check that API key is not expired and has the basemaps privilege assigned.');
```

Reply:
I appreciate the feedback everyone 🙏 I was also thinking of how to handle cases for expired keys and adding to the warn statement didn't even cross my mind 😭😂